tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -193,10 +193,16 @@ public function sendShareInvitation(string $user, TableNode $table, string $file
 			$sharees = array_map('trim', explode(',', $rows['sharee']));
 			$shareTypes = array_map('trim', explode(',', $rows['shareType']));
 
-			foreach ($sharees as $sharee) {
+			foreach ($sharees as $index => $sharee) {
+				$shareType = $shareTypes[$index];
+				$shareeId = "";
+				if ($shareType === "user") {
+					$shareeId = $this->featureContext->getAttributeOfCreatedUser($sharee, 'id');
+				} elseif ($shareType === "group") {
+					$shareeId = $this->featureContext->getAttributeOfCreatedGroup($sharee, 'id');
+				}
 				// for non-exiting group or user, generate random id
```

Comment:
```suggestion
				// for non-existing group or user, generate random id
```
some day this typo could be fixed, but not related to this PR :)